lib/src/model/library.dart

```diff
@@ -652,4 +653,26 @@ class Library extends ModelElement with Categorization, TopLevelContainer {
     return (_allCanonicalModelElements ??=
         allModelElements.where((e) => e.isCanonical).toList());
   }
+
+  Map<String, CommentReferable> _referenceChildren;
+  @override
+  // TODO(jcollins-g): This should take the import/export graph
+  // and resulting namespace into account.
+  Map<String, CommentReferable> get referenceChildren {
+    if (_referenceChildren == null) {
+      _referenceChildren = {};
```

Comment:
This could also be a collection element thing, but it might read better how it is, depending on how dartfmt forces newlines after `for (var e in constants)`, etc.